htmlmaker_postprocessing.js

```diff
@@ -95,6 +106,83 @@ fs.readFile(file, function editContent (err, contents) {
     $(this).after(match[4]);
   });
 
+  // remove Section-Blank-Page sections
+  var sectionBlankPage = $("section.blankpage");
+  sectionBlankPage.remove();
+
+  //// Strip pageBreaks preceding Section starts: Part 1 (of 2):
+  // catch & remove any page break directly preceding Section Starts
+  //  (nothing should be outside of a seciton block, but just in case)
+  var leadingPageBreak = $("section").prev(".PageBreakpb");
+  // cycle through each Page Break para directly preceding a Section Start
+  leadingPageBreak.each(function() {
+    // capture any Page Breaks directly preceding the selected one
+    var consecutiveLeadingPBs = $(this).prevUntil(":not(.PageBreakpb)").addBack();
+    consecutiveLeadingPBs.remove();
+  });
+
+  //// Strip pageBreaks preceding Section starts: Part 2 (of 2):
```

Comment:
I had issues with a couple of my initial choices of selectors here, a couple of things I expected to work just did not work, for no reason I could figure out. I also split them up some b/c I was doing that loop; so I'll try things again without.